docs/plugins/spotify.rst

```diff
@@ -37,6 +58,24 @@ Command-line options include:
 * ``--show-failures`` or ``-f``: List the tracks that did not match a Spotify
   ID.
 
+A Spotify ID or URL may also be provided to the ``Enter release ID:``
+prompt during import::
+
+    Enter search, enter Id, aBort, eDit, edit Candidates, plaY? i
+    Enter release ID: https://open.spotify.com/album/2rFYTHFBLQN3AYlrymBPPA
+    Tagging:
+        Bear Hands - Blue Lips / Ignoring the Truth / Back Seat Driver (Spirit Guide) / 2AM
+    URL:
+        https://open.spotify.com/album/2rFYTHFBLQN3AYlrymBPPA
+    (Similarity: 88.2%) (source, tracks) (Spotify, 2019, Spensive Sounds)
+     * Blue Lips (feat. Ursula Rose)   -> Blue Lips (feat. Ursula Rose) (source)
+     * Ignoring the Truth              -> Ignoring the Truth (source)
+     * Back Seat Driver (Spirit Guide) -> Back Seat Driver (Spirit Guide) (source)
+     * 2AM                             -> 2AM (source)
+    [A]pply, More candidates, Skip, Use as-is, as Tracks, Group albums,
+    Enter search, enter Id, aBort, eDit, edit Candidates, plaY?
```

Comment:
I'm not sure having the whole candidate display is necessary here. Maybe the just the first two lines would be enough to get the idea across?